install.sh

```diff
@@ -17,6 +22,16 @@ debian_distribution_name=""
 sfx_ingest_url="https://ingest.signalfx.com"
 dimensions=""
 input_collectd=""
+# Number of seconds system clock can be out of sync from NTP server.
+time_sync_max_drift=10
+
+is_bsd() {
```

Comment:
Yeah I thought about that but `determine_os` makes more specific distinctions. Imagine we add FreeBSD support then `determine_os` needs to know exactly what the host OS is since the packaging will be different. Whereas this is just about BSD vs. GNU (assuming date is similar on FreeBSD we don't need it to be any more specific).